functions/go/set-annotations/annotations_transformer.go

```diff
@@ -21,6 +24,18 @@ type plugin struct {
 	FieldSpecs []types.FieldSpec `json:"fieldSpecs,omitempty" yaml:"fieldSpecs,omitempty"`
 	// AdditionalAnnotationFields is used to specify additional fields to add annotations.
 	AdditionalAnnotationFields []types.FieldSpec `json:"additionalAnnotationFields,omitempty" yaml:"additionalAnnotationFields,omitempty"`
+	// Results are the results of applying annotations
+	Results []*Result
+}
+
+// Result holds result of search and replace operation
+type Result struct {
+	// FilePath is the file path of the matching field
```

Comment:
copy/paste maybe. Update these for set-annotations

Reply:
Good catch, done.